Probesys/bileto
# fix: Force requester and assignee to be part of the lists

## Related issue(s)



N/A

## How to test manually



- create a ticket and set a requester + assignee
- check they are correctly set
- update the requester and assignee of the ticket
- check they are correctly set

## Checklist
